src/main/java/com/esri/arcgisruntime/toolkit/skins/CompassSkin.java

```diff
@@ -87,9 +87,9 @@ public CompassSkin(com.esri.arcgisruntime.toolkit.Compass control) {
       .and(controlHeadingProperty.isEqualTo(0.0, HEADING_TOLERANCE)
       .or(controlHeadingProperty.isEqualTo(360.0, HEADING_TOLERANCE))));
     hiddenProperty.addListener(observable -> {
-      // when the hidden property changes schedule to execute a fade in/out - having a delay prevents the compass from
```

Comment:
I don't think "execute" and "executor" are on the inclusive language avoid list.

Reply:
they are not, but why not change it? imo it is as bad as kill, hang, terminate etc

Reply:
I don't know why they aren't on the list but one possibility is execute has multiple definitions that have nothing to do with violence e.g. execute a task. I believe executor is also unrelated to execution in the violent sense.

Reply:
thats also true of other terms on the list

Reply:
I think we are better sticking to the guidance we have been given on what to replace and what to replace it with.

It doesn't make sense to replace "executor" since it isn't on the list and the word still appears multiple time in
```
private final ScheduledExecutorService scheduledService = Executors.newScheduledThreadPool(0);
```